stdlib/public/core/StringBridge.swift

```diff
@@ -250,11 +250,6 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     }
   }
 
-  @objc
-  func _fastCharacterContents() -> UnsafeMutablePointer<UInt16>? {
-    return _core.elementWidth == 2 ? _core.startUTF16 : nil
-  }
-
   //
```

Comment:
@milseman originally there was no foundation overlay :-). Given that these methods are only looked up by objc_msgSend, it seems plausible to me that they could live in the overlay.